client/me/notification-settings/blogs-settings/header.jsx

```diff
@@ -39,23 +38,35 @@ class BlogSettingsHeader extends PureComponent {
 
 	getLegend = () => {
 		const { settings } = this.props;
-		const filteredSettings = omit( settings, [
-			'blog_id',
-			'devices',
-			'email.achievement',
-			'email.store_order',
-			'email.scheduled_publicize',
-			'timeline.store_order',
-		] );
+		const {
+			blog_id,
+			devices,
+			'email.achievement': emailAchievement,
+			'email.store_order': emailStoreOrder,
+			'email.scheduled_publicize': emailScheduledPublicize,
+			'timeline.store_order': timelineStoreOrder,
+			...filteredSettings
```

Comment:
Nice, I hadn't considered using this approach for `omit` before. I guess the downside is adding a few extra variables to memory, but that's probably not going to have an impact

Reply:
Yup, those are pretty cheap.